pandas/tests/indexes/interval/test_interval.py

```diff
@@ -497,6 +497,23 @@ def test_get_loc_interval(self):
         pytest.raises(KeyError, self.index.get_loc,
                       Interval(-1, 0, 'left'))
 
+    # To be removed, replaced by test_interval_new.py (see #16316, #16386)
+    def test_get_loc_exact(self):
+        # GH19353
+        assert self.index.get_loc_exact(Interval(0, 1)) == 0
+        with pytest.raises(KeyError):
+            self.index.get_loc_exact(1)
+        with pytest.raises(KeyError):
+            self.index.get_loc_exact(Interval(0, 1, 'left'))
+        with pytest.raises(KeyError):
+            self.index.get_loc_exact(Interval(0, 0.5))
+        with pytest.raises(KeyError):
+            self.index.get_loc_exact(Interval(2, 3))
+        with pytest.raises(KeyError):
+            self.index.get_loc_exact(Interval(-1, 0, 'left'))
+        # See #19353#issuecomment-364295029
```

Comment:
what is this last test?

Reply:
this tests if the caching issue showed by @jschendel exists.

Reply:
ok then write that as a comment. an issue reference is good, but a comment + reference even better. add blank lines in between sections of tests.